src/app.js

```diff
@@ -0,0 +1,21 @@
+const scoreListElement = document.getElementById('scores');
+
+const addScoreToList = async (score) => {
+  await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/QmA8PefwGdv5BMnpfXEQ/scores/', {
+    method: 'POST',
```

Comment:
- [ ] Great work so far 👍 but I have seen that you are writing your whole API URL at different places in the app 🤔 right now it is not a problem but suppose you use it at 10 different places and if you need to change it then it would be very difficult for you to do it because there might be mistakes happen while changing them so the ultimate solution for this problem is that you can store the API in a different file and import that API anywhere in your app where you wanna use it 😊
- [ ] Secondly, it would be better if you do this because it is according to the DRY principal which says don't repeat yourself 👍